doc/api/api_changes/2017-12-12-JMK.rst

```diff
@@ -0,0 +1,4 @@
+`~.hist2d` now uses `~.pcolormesh` instead of `~.pcolorfast`
+------------------------------------------------------------
+
+`~.hist2d` now uses `~.pcolormesh` instead of `~.pcolorfast`.
```

Comment:
explicitly mention that this changes the return type?

Reply:
And that the rationale is to provide support for log axes.

Reply:
But `AxesImage now behaves correctly on log scales.

Reply:
OK, so we don't need this anymore? I didn't check recently that log scales now work.  

OTOH, Somewhat prefer pcolormesh, but thats just out of familiarity.